README.md

```diff
@@ -55,9 +51,9 @@ LDK/`rust-lightning` is a generic library which allows you to build a Lightning
 node without needing to worry about getting all of the Lightning state machine,
 routing, and on-chain punishment code (and other chain interactions) exactly
 correct. Note that `rust-lightning` isn't, in itself, a node. There are various
-working/in progress demos which could be used as a node today, but if you "just"
-want a generic Lightning node, you're almost certainly better off with [Core
-Lightning](https://github.com/ElementsProject/lightning) or
+nodes built using LDK such as [Sensei](https://l2.technology/sensei), but if you
+"just" want a generic Lightning node, you're almost certainly better off with
+[Core Lightning](https://github.com/ElementsProject/lightning) or
```

Comment:
Yeah Sensei is still documented as "beta", though it would be good opportunity to mark it as "call for beta users or testers". Other Lightning nodes can be just pointed out as "production-ready". Or we can also scrap them out.

Reply:
Since we have Sensei approaching production-readyness, I think now we can scrap out references to other Lightning implementations. For a Lightning newcomer, all the implementations could be references at the BOLT-level, somehow in `CONTRIBUTING.md`